app/renuo_upload_signing.rb

```diff
@@ -44,6 +45,18 @@ class RenuoUploadSigning < Sinatra::Base
     body 'up'
   end
 
+  delete '/delete_file' do
+    set_response_headers
```

Comment:
I remarked right now, that the first 3 lines of the methods "generate_policy", "list_files" and "/delete_file" are the same. Could we dry it up?